packages/common/src/errors.spec.js

```diff
@@ -0,0 +1,51 @@
+import { AccountsError } from './errors';
+
+const error = new AccountsError(
+  'Validation Error',
+  {
+    username: 'user',
+  },
+  'ACCOUNTS:1',
+);
+
+const throws = () => {
+  throw error;
+};
+
+describe('AccountsError class', () => {
+  it('should be throwable', () => expect(throws).toThrow());
+
+  it('should carry relevant info', () => {
+    try {
+      throws();
+    } catch (e) {
+      expect(e.message).toBe('Validation Error');
+      expect(e.loginInfo).toMatchObject({ username: 'user' });
+      expect(e.errorCode).toBe('ACCOUNTS:1');
+      expect(e.epochTime).toBeDefined();
+    }
+  });
+
+
+  // NB: For some reason it seems jest have different Error class
```

Comment:
Maybe an approach leveraging http://facebook.github.io/jest/docs/api.html#extending-jest-matchers could be possible to produce something alone the lines of `expect(function).toThrowAccountsError()`.